experiments/run_experiment.py

```diff
@@ -1,20 +1,30 @@
-"""A mainfile for running MRMC experiments.
+"""A mainfile for running recourse experiments.
```

Comment:
Suggest to rename file run_recourse_experiment.py so it's a little clearer that it is a generic runner for all recourse methods.